AirCasting/AirCastingApp.swift

```diff
@@ -62,10 +62,13 @@ struct AirCastingApp: App {
         }.onChange(of: scenePhase) { newScenePhase in
             switch newScenePhase {
             case .active:
+                persistenceController.uiSuspended = false
                 appBecameActive.send()
             case .background:
+                persistenceController.uiSuspended = true
                 break
             case .inactive:
+                persistenceController.uiSuspended = true
                 break
```

Comment:
Don't You want to merge those two into one, as they have the same condition?

Reply:
good catch!